helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -526,6 +528,7 @@ logsCollection:
     index: ""
 
   checkpointPath: "/var/addon/splunk/otel_pos"
+  persistentStorageDirectory: "persistent_storage"
```

Comment:
I don't think we should put it in checkpointing subdirectory. Maybe `persistentQueueStoragePath: /var/addon/splunk/persistent_queue` ?

Reply:
It was also considered.I will implement this feedback in the next year :). Marry Christmas and Happy New Year!!!

Reply:
Merry Christmas and Happy New Year! :)